src/EFCore.PG/Extensions/NpgsqlIndexBuilderExtensions.cs

```diff
@@ -47,5 +52,54 @@ public static IndexBuilder ForNpgsqlHasOperators([NotNull] this IndexBuilder ind
 
             return indexBuilder;
         }
+
+        /// <summary>
+        /// Adds an INCLUDE clause to the index definition with the specified property names.
+        /// This clause specifies a list of columns which will be included as a non-key part in the index.
+        /// </summary>
+        /// <remarks>
+        /// https://www.postgresql.org/docs/current/sql-createindex.html
+        /// </remarks>
+        /// <param name="indexBuilder"> The builder for the index being configured. </param>
+        /// <param name="propertyNames"> An array of property names to be used in INCLUDE clause. </param>
+        /// <returns> A builder to further configure the index. </returns>
+        public static IndexBuilder ForNpgsqlInclude([NotNull] this IndexBuilder indexBuilder, [CanBeNull] params string[] propertyNames)
```

Comment:
nit: Add `[ItemNotNull]` for the `propertyNames`.
nit: Consider wrapping arguments.